packages/components/src/palette-edit/index.tsx

```diff
@@ -602,9 +602,13 @@ export function PaletteEdit( {
 								disableCustomColors
 							/>
 						) ) }
-				</>
+				</PaletteEditContents>
+			) }
+			{ ! hasElements && emptyMessage && (
+				<PaletteEditContents>
+					{ emptyMessage }
+				</PaletteEditContents>
```

Comment:
Prettier will complain about this, so we'll need to fix it before merging:

```suggestion
				<PaletteEditContents>{ emptyMessage }</PaletteEditContents>
```